apps/dapp-console/package.json

```diff
@@ -4,14 +4,15 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build --no-lint",
```

Comment:
this prevents next from using it's own linting rules and ignoring the ones we have setup